oat-sa/extension-tao-itemqti
# Feature/tca 389/item import lock ttl

Pull request summary
 
Related to : https://oat-sa.atlassian.net/browse/TCA-389
 
Added configuration parameter with TTL of the locker for the item import process
 
#### How to test
 
Set configured parameter TTL to the 1, run item importing (item has a lot of media files, that can't be imported in 1 second), check the log that has record, that locker was released before item was imported (so theoretically someone can ask the item files that weren't imported at this time)

#### Dependencies

Companion PR
- [ ] https://github.com/oat-sa/extension-tao-act/pull/1346
